packages/preprocess/src/process-frontmatter/handle-og.svelte

```diff
@@ -1,33 +1,33 @@
 <!-- 
-        MDSvex comes in handy here because it takes frontmatter and shoves it into the metadata object.
-        This means that all we need to do is build out the expected page metadata
-    -->
+    MDSvex comes in handy here because it takes frontmatter and shoves it into the metadata object.
+    This means that all we need to do is build out the expected page metadata
+-->
 {#if typeof metadata !== "undefined" && (metadata.title || metadata.og?.title)}
```

Comment:
I am curious why not if `metadata` !== `undefined`

Reply:
In some cases; if metadata is _never_ assigned (e.g. not even set to undefined), then referencing it at all causes an error, doing it this way ensures that it will not break even if metadata doesn't exist at all.